editors/code/package.json

```diff
@@ -546,6 +546,11 @@
                     "minimum": 0,
                     "maximum": 255
                 },
+                "rust-analyzer.cargo.allTargets": {
+                    "markdownDescription": "Pass `--all-targets` to cargo invocation. Overridden by `#rust-analyzer.check.allTargets#`\nwhen the latter is set.",
```

Comment:
Is it truly overridden? It seems to me like it is only partially overridden. Specifically, the build script / proc macro build will always use the value of `rust-analyzer.cargo.allTargets` even if `rust-analyzer.check.allTargets` is set.

If that's correct then the docs are wrong or at least misleading.

Reply:
Indeed, I misread this.

Reply:
I should have precised "for check calls" maybe

Reply:
I adjusted it in https://github.com/rust-lang/rust-analyzer/pull/16988 making it the same as the docs for `allFeatures`